crates/flux-middle/src/queries.rs

```diff
@@ -193,6 +193,7 @@ pub struct Providers {
     pub item_bounds:
         fn(GlobalEnv, MaybeExternId) -> QueryResult<rty::EarlyBinder<List<rty::Clause>>>,
     pub sort_decl_param_count: fn(GlobalEnv, FluxId<MaybeExternId>) -> usize,
+    pub no_panic: fn(GlobalEnv, MaybeExternId) -> QueryResult<bool>,
```

Comment:
Since you are inlining the implementation, you don't need a provider. I'll mark all the parts you can remove.

```suggestion
```